addons/kaptain/0.x/kaptain-2.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: ClusterAddon
+metadata:
+  name: kaptain
+  labels:
+    kubeaddons.mesosphere.io/name: kaptain
+    kubeaddons.mesosphere.io/hack-requires-defaultstorageclass: "true"
+  annotations:
+    kudo.kubeaddons.mesosphere.io/create-namespace: "true"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.0.1-2"
+    appversion.kubeaddons.mesosphere.io/kaptain: "1.0.1"
+spec:
+  namespace: kubeflow
+  kubernetes:
+    minSupportedVersion: v1.16.0
+  requires:
```

Comment:
since Kaptain components include several `ServiceMonitor` resources, should we also mention Prometheus addon as a requirement here?

Reply:
good catch, nfs provisioner is actually required too.